src/codegen/clean-generated.ts

```diff
@@ -0,0 +1,39 @@
+import fs from 'fs';
+import glob from 'glob';
+import path from 'path';
+import rimraf from 'rimraf';
+import { promisify } from 'util';
+import yargs from 'yargs';
```

Comment:
I'm not quite sure I like introducing a dependency for this simple use-case when `process.argv.includes('--empty-only')` would do the same thing. Using a library for parsing arguments is useful when the argument list becomes lengthy, but here it's just one boolean arg.